src/l500/l500-options.h

```diff
@@ -25,12 +25,46 @@ namespace librealsense
         get_current = 0,
         get_min = 1,
         get_max = 2,
-        get_step = 3
+        get_step = 3,
+        get_default = 4
     };
 
-    class l500_hw_options : public option
+class l500_options;
+
+// On old FW versions the way to get the default values of the hw commands is to
+// reset hw commands current values -1 and than get the current values
+// on some of the old FW versions there is a bug that we must reset hw commands
+// before setting the digital gain, otherwise its not updates the current with default values
+// in digital_gain_option class we override the set_with_no_signal that called when 
+// changing preset and reset hw commands before setting the digital gain as WA to this bug
+// we still have a limit on the scenario that user change digital gain manually (not from preset)
+// we won't get the correct default values
```

Comment:
This description is good, but needs to be moved to the implementation of `set_with_no_signal()`